src/Geno/C++/Application.cpp

```diff
@@ -62,9 +62,9 @@ void Application::NewWorkspace( std::filesystem::path Location, std::string Name
 {
 	CloseWorkspace();
 
-	Workspace& rWorkspace    = m_CurrentWorkspace.emplace( std::move( Location ) );
-	rWorkspace.m_Name        = std::move( Name );
-	rWorkspace.m_BuildMatrix = BuildMatrix::PlatformDefault();
+	m_CurrentWorkspace                = new Workspace( std::move( Name ), std::move( Location ) );
+	m_CurrentWorkspace->m_BuildMatrix = BuildMatrix::PlatformDefault();
+	m_CurrentWorkspace->m_ExpandNode  = true;
```

Comment:
`m_CurrentWorkspace` should not be heap-allocated. It can still be an std::optional even though it derives from an interface (abstract class)